### PR TITLE
[ja] Translate reference/glossary/addons.md & kubeadm.md into Japanese

### PR DESCRIPTION
**Pull request by kujiraitakahiro**
This is a Feature Request

What would you like to be added

Translate [content/en/docs/reference/glossary/addons.md](https://kubernetes.io/ja/docs/reference/glossary/?tool=true#term-addons) &  [content/en/docs/reference/glossary/kubeadm.md](https://kubernetes.io/ja/docs/reference/glossary/?tool=true#term-kubeadm) into Japanese.

Why is this needed
This page has not been translated.

Comments

/language ja
/assign

### CODE REVIEW

File: content/ja/docs/reference/glossary/addons.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Add-ons
+id: addons
+date: 2019-12-15
+full_link: /docs/concepts/cluster-administration/addons/
```
Comment (atoato88) [inline content/ja/docs/reference/glossary/addons.md:5]: We already have a `ja` page on this page, should link to `ja` page.
```suggestion
full_link: /ja/docs/concepts/cluster-administration/addons/
```

File: content/ja/docs/reference/glossary/addons.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  Kubernetesの機能を拡張するリソース。
+
+aka:
+tags:
+- tool
+---
+  Kubernetesの機能を拡張するリソース。
+
+<!--more-->
+[Installing addons](/docs/concepts/cluster-administration/addons/)では、クラスターのアドオン使用について詳しく説明し、いくつかの人気のあるアドオンを列挙します。
```
Comment (atoato88) [inline content/ja/docs/reference/glossary/addons.md:16]: ditto.
```suggestion
[Installing addons](/ja/docs/concepts/cluster-administration/addons/)では、クラスターのアドオン使用について詳しく説明し、いくつかの人気のあるアドオンを列挙します。
```

File: content/ja/docs/reference/glossary/kubeadm.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Kubeadm
+id: kubeadm
+date: 2018-04-12
+full_link: /docs/reference/setup-tools/kubeadm/
```
Comment (atoato88) [inline content/ja/docs/reference/glossary/kubeadm.md:5]: ditto.
```suggestion
full_link: /ja/docs/reference/setup-tools/kubeadm/
```

**Review by atoato88 — COMMENTED** [review]
Almost LGTM.
I commented about letting `full_link` link to `ja` pages.

**Comment by atoato88** [comment]
Almost LGTM.

Could you squash multiple commits to one commit?

**Comment by atoato88** [comment]
Thank you for quick update!! 😄

File: content/ja/docs/reference/glossary/kubeadm.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: Kubeadm
+id: kubeadm
+date: 2018-04-12
+full_link: /ja/docs/reference/setup-tools/kubeadm/
+short_description: >
+  Kubernetesを迅速にインストールし、安全なクラスタをセットアップするためのツール。
```
Comment (t-inu) [inline content/ja/docs/reference/glossary/kubeadm.md:7]:
```suggestion
  Kubernetesを迅速にインストールし、安全なクラスターをセットアップするためのツール。
```

File: content/ja/docs/reference/glossary/kubeadm.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+aka: 
+tags:
+- tool
+- operation
+---
+ Kubernetesを迅速にインストールし、安全なクラスタをセットアップするためのツール。
```
Comment (t-inu) [inline content/ja/docs/reference/glossary/kubeadm.md:14]:
```suggestion
 Kubernetesを迅速にインストールし、安全なクラスターをセットアップするためのツール。
```

File: content/ja/docs/reference/glossary/addons.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  Kubernetesの機能を拡張するリソース。
+
+aka:
+tags:
+- tool
+---
+  Kubernetesの機能を拡張するリソース。
+
+<!--more-->
+[Installing addons](/ja/docs/concepts/cluster-administration/addons/)では、クラスターのアドオン使用について詳しく説明し、いくつかの人気のあるアドオンを列挙します。
```
Comment (t-inu) [inline content/ja/docs/reference/glossary/addons.md:16]:
```suggestion
[アドオンのインストール](/ja/docs/concepts/cluster-administration/addons/)では、クラスターのアドオン使用について詳しく説明し、いくつかの人気のあるアドオンを列挙します。
```
It's better to use Japanese page title.

File: content/ja/docs/reference/glossary/kubeadm.md (hunk 5 above)
Comment (t-inu) [reply content/ja/docs/reference/glossary/kubeadm.md:14]: It seems this "クラスタ" has not changred to "クラスター" yet.

**Comment by atoato88** [comment]
/assign @nasa9084 
Could you review this?

**Pull request merged**